diff --git a/characters/automatas/snack/snack_vending_machine.tscn b/characters/automatas/snack/snack_vending_machine.tscn
index f1c17b0..7638ece 100644
--- a/characters/automatas/snack/snack_vending_machine.tscn
+++ b/characters/automatas/snack/snack_vending_machine.tscn
@@ -1,4 +1,4 @@
-[gd_scene load_steps=8 format=3 uid="uid://40q6al2c4n4n"]
+[gd_scene load_steps=9 format=3 uid="uid://40q6al2c4n4n"]

 [ext_resource type="Script" uid="uid://b1efgwx06mllx" path="res://characters/automatas/automata.gd" id="1_0t7m3"]
 [ext_resource type="Script" uid="uid://gscj3hnm5oor" path="res://items/item.gd" id="2_vdv0g"]
@@ -6,6 +6,7 @@
 [ext_resource type="Resource" uid="uid://brecpjrathqvd" path="res://items/consumables/potato.tres" id="4_lsxso"]
 [ext_resource type="Texture2D" uid="uid://bk62howen4nad" path="res://characters/automatas/snack/assets/snack_vending_machine.png" id="4_qdfgr"]
 [ext_resource type="Resource" uid="uid://dyefx1h80soau" path="res://items/consumables/cigarette.tres" id="5_ku304"]
+[ext_resource type="Resource" uid="uid://ckynd8kmolf6x" path="res://items/weapons/slowpoke_tail.tres" id="6_1mrxr"]

 [sub_resource type="RectangleShape2D" id="RectangleShape2D_1eypk"]
 size = Vector2(20, 7)
@@ -13,7 +14,7 @@ size = Vector2(20, 7)
 [node name="SnackVendingMachine" type="StaticBody2D"]
 collision_layer = 5
 script = ExtResource("1_0t7m3")
-shop_inventory = Array[ExtResource("2_vdv0g")]([ExtResource("3_15wnu"), ExtResource("4_lsxso"), ExtResource("5_ku304"), null, null, null, null, null])
+shop_inventory = Array[ExtResource("2_vdv0g")]([ExtResource("3_15wnu"), ExtResource("4_lsxso"), ExtResource("5_ku304"), ExtResource("6_1mrxr"), null, null, null, null])
 shop_name = "Snackautomat"

 [node name="Sprite2D" type="Sprite2D" parent="."]
diff --git a/items/consumables/potato.tres b/items/consumables/potato.tres
index ee19399..66da5b9 100644
--- a/items/consumables/potato.tres
+++ b/items/consumables/potato.tres
@@ -7,7 +7,7 @@
 [sub_resource type="Resource" id="Resource_pp2si"]
 script = ExtResource("2_ki2lp")
 max_health = 0
-health = 5
+health = 500
 armor = 0
 initiative = 0
 dodge_chance = 0
@@ -28,7 +28,7 @@ drug_level = 0
 script = ExtResource("1_whwh5")
 effect_duration = 0
 stats = SubResource("Resource_pp2si")
-price = 5
+price = 0
 name = "Kartoffel"
 description = "Eine normale rohe Kartoffel, für die nicht normalen Kartoffeln in der U-Bahn. "
 texture = ExtResource("3_j3t3a")
